google-cloud-pub-sub-grpc/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/grpc/scaladsl/GooglePubSub.scala

```diff
@@ -94,21 +94,38 @@ object GooglePubSub {
       }
       .mapMaterializedValue(_.flatMap(identity)(ExecutionContexts.sameThreadExecutionContext))
 
+  /**
+   * Create a flow that accepts consumed message acknowledgements.
+   */
+  def acknowledgeFlow(): Flow[AcknowledgeRequest, AcknowledgeRequest, NotUsed] =
+    Flow
+      .setup { (mat, attr) =>
+        Flow[AcknowledgeRequest]
+          .mapAsync(1)(
+            req =>
+              subscriber(mat, attr).client
+                .acknowledge(req)
+                .map(_ => req)(mat.executionContext)
+          )
+      }
+      .mapMaterializedValue(_ => NotUsed)
```

Comment:
Maybe there's an opportunity here to compose all the acknowledge sink/flow and java/scala DSL methods from this to DRY up the impls. It could be refactored into a private method that takes `parallelism` to support the deprecated overloads.

Reply:
The gRPC generated types are different for the Java DSL and the Scala DSL. That's not easy to see.